clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -141,6 +141,10 @@
                                                            + "The value must be set lower than <code>session.timeout.ms</code>, but typically should be set no higher "
                                                            + "than 1/3 of that value. It can be adjusted even lower to control the expected time for normal rebalances.";
 
+    public static final String DEFAULT_API_TIMEOUT_MS_CONFIG = "default.api.timeout.ms";
+    public static final String DEFAULT_API_TIMEOUT_MS_DOC = "Specifies the timeout (in milliseconds) for client APIs that could block. " +
```

Comment:
We might need to generalize this description a little bit. The AdminClient APIs do not block.

Reply:
Simply dropping `could block` is a good option?

Reply:
Yeah, that sounds fine.